src/components/App.js

```diff
@@ -20,6 +20,7 @@ export class App extends Component {
   }
 
   componentDidMount = () => {
+    document.title = "Secure Sign";
```

Comment:
I think this will work but, you might be better off changing `<title>React App</title>` in the index.html. Also, if you search through the project `React App` will appear in the manifest. I'm not sure what that does but for consistency we should update 'em all.

Reply:
Resolved.